### DISCUSSION TITLE
SMTP server "Permission denied - bind (::) port 25"

### DISCUSSION THREAD

**Discussion by sandrovenezuela** [postalserver/postal#2836] [Help with using Postal]
Hello,

There was a problem on just one of my Postal servers causing the SMTP service to terminate because it was unable to create communication with port 25.

Error presented is "Permission denied - bind (::) port 25".

There is no other service using port 25 and the solution to the problem was to change the net.ipv4.ip_unprivileged_port_start parameter equal to zero through the sysctl command.

I believe that the person responsible for this error was the docker composer plugin.

I say this because it is the only package with a different version that I have among the other Postal servers.

All servers use Debian 11, Postal 2.1.4 and the version of the docker-composer-plugin package is 2.24.5-1, with the only exception being the server that presented the problem, which uses version 2.24.6-1.

Has anyone ever had this type of problem?

I still don't understand why the problem happened now since the server has been in production for some time. Not to mention, as I said, the other servers are working normally.

Any help will be welcome.

Thanks!

**Comment by willpower232** [comment]
I normally wouldn't recommend using the docker compose plugin package as it is usually out of date but judging by https://github.com/docker/compose/releases, this one has been kept up to date.

I will say that I use 2.23.0 currently and rarely update because there is often a bug present in the new version and it has put me off regular updating. Perhaps you could download a working version from github as part of your setup?

**Reply by sandrovenezuela** [reply]
Hi Will,

I install Docker using packages for ease of maintenance later, but situations like this can actually occur during an update.

I'll check to downgrade to the previous version of the docker composer package.

In parallel, I will research whether this version 2.24.6 really introduced this type of restriction and what are the possible solutions.

Thanks.
